test/isSymbol.js

```diff
@@ -0,0 +1,30 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('isSymbol', function() {
+  context('given a Symbol as value', function() {
+    const PolyfilledSymbol = function Symbol(val) {
+      this.val = val;
+    };
+
+    it('should return true', function() {
+      assert.isTrue(RA.isSymbol(Symbol('any')));
+      assert.isTrue(RA.isSymbol(Symbol(1)));
+      assert.isTrue(RA.isSymbol(Symbol.iterator));
+      assert.isTrue(RA.isSymbol(new PolyfilledSymbol(1)));
+    });
+  });
+
+  context('given anything else as value', function() {
+    it('should return false', function() {
+      assert.isFalse(RA.isSymbol(undefined));
+      assert.isFalse(RA.isSymbol(null));
+      assert.isFalse(RA.isSymbol(() => {}));
+      assert.isFalse(RA.isSymbol(NaN));
+      assert.isFalse(RA.isSymbol(true));
+      assert.isFalse(RA.isSymbol('string'));
+      assert.isFalse(RA.isSymbol(1));
+    });
+  });
+});
```

Comment:
We need to add one additional test which is testing that the function is curried:
```js
  it('should support placeholder to specify "gaps"', function() {
    const isSymbol = RA.isSymbol(R.__);

    assert.isFalse(isSymbol('abc'));
  });

Reply:
Done dc6d287c652e63f8ac31c3511ac97eb238bcf7a4